Makefile

```diff
@@ -8,3 +8,4 @@ prelaunch-3:
 regen-1:
 	go run . build-genesis regen-1
 	mv -f regen-1/genesis.json regen-1/genesis-prelaunch.json
```

Comment:
Why not just use genesis.json now instead of genesis-prelaunch?

Reply:
We'll use `genesis.json` for final genesis with gentxs. So, let's use `genesis-prelaunch`

Reply:
I agree that it'd be nice to not have `genesis-prelaunch.json` checked in to the repo now that we have the final `genesis.json`. IMO either we should remove `genesis-prelaunch.json` as a last step in this makefile, or just drop this `mv -f regen-1/genesis.json regen-1/genesis-prelaunch.json` line, and then overwrite `genesis.json` in the `gen-genesis.sh` script.

Reply:
Up to you guys